test/lib/formatHEC.test.js

```diff
@@ -0,0 +1,29 @@
+const chai = require('chai');
+chai.should();
+const expect = chai.expect;
+
+import formatHEC from '../../dist/lib/formatHEC';
+
+describe('FormatHEC', () => {
+
+	it('should exist', () => {
+		formatHEC.should.exist;
+	});
+
+	it('should format the data', () => {
+		const options = {
+			level: 'info',
+			message: 'Hello world',
+			meta: {
+				event: 'test'
+			}
+		};
```

Comment:
Let's get a few more test cases in here.

Can `message` ever be an object itself?

Can `meta` ever be a string?